ci: add wait-timeout,wait flag to minikube start

Madhu-1 opened the pull request:
minikube waits for 6 minutes for the minikube or host to be ready, due to resource issues sometimes
the host/minikube might take a longer time to start the cluster. Increase the timeout to 10m by default. Added wait flag with default `all` to wait for all services to be ready.

updates #1969

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

Madhu-1 committed:
ci: add wait-timeout flag to minikube start

minikube waits for 6 minutes for the minikube or
host to be ready, due to resource issue sometimes
the host/minikube might take longer time to start
the cluster. Increase the timeout to 10m by default

updates #1969

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

nixpanic reviewed (commented):
I wonder if it is needed to wait for other components?

```
      --wait strings                      comma separated list of Kubernetes components to verify and wait for after starting a cluster. defaults to "apiserver,system_pods", available options: "apiserver,system_pods,default_sa,apps_running,node_ready,kubelet" . other acceptable values are 'all' or 'none', 'true' and 'false' (default [apiserver,system_pods])
```

In general minikube becomes ready, so maybe wait for `node_ready` too?

Madhu-1 commented:
> I wonder if it is needed to wait for other components?
> 
> ```
>       --wait strings                      comma separated list of Kubernetes components to verify and wait for after starting a cluster. defaults to "apiserver,system_pods", available options: "apiserver,system_pods,default_sa,apps_running,node_ready,kubelet" . other acceptable values are 'all' or 'none', 'true' and 'false' (default [apiserver,system_pods])
> ```
> 
> In general minikube becomes ready, so maybe wait for `node_ready` too?

How about `all` ?

nixpanic commented:
> How about `all` ?

Sure, we expect everything to be available before we start running the tests. I don't know if we need everything, but I guess it won't hurt either.

Madhu-1 committed:
ci: add wait flag to minikube start

add wait flag to minikube start to wait
for all components to be ready by default.

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>
